src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -24,7 +15,42 @@ interface ItemProps {
 
 const Item: React.FC<ItemProps> = props => {
   const { item } = props
-  if (item.__typename === "Artwork") {
+  const itemType = item.__typename
+
+  const getImage = item => {
+    if (itemType === "Artwork") {
+      return item.image?.url
+    } else if (itemType === "Show") {
+      return item.coverImage?.url
+    } else {
+      return null
+    }
+  }
+
+  const itemDetails = item => {
```

Comment:
The styling for "Artwork" and "Show" really only diverge when it comes to the item details so I pulled that out into a separate function.